catalog/spatial/csw/spatial-csw-source/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -78,6 +98,59 @@
             <property name="eventServiceAddress" value=""/>
             <property name="registerForEvents" value="false"/>
 
+            <property name="schemaTransformerManager" ref="metacardTransformerManager"/>
+            <cm:managed-properties persistent-id="" update-strategy="component-managed"
+                                   update-method="refresh"/>
+        </cm:managed-component>
+    </cm:managed-service-factory>
+
+    <cm:managed-service-factory
+            id="org.codice.ddf.spatial.ogc.csw.catalog.source.CswCatalogStore.id"
```

Comment:
They were very similar except a few fields. There is currently no use case for just a straight csw store so I removed this MSF